lib/internal/tls.js

```diff
@@ -24,5 +24,5 @@ function parseCertString(s) {
 }
 
 module.exports = {
-  parseCertString
```

Comment:
Why the comma is introduced?

Reply:
It's unintentionally introduced(my personal code style :-)). I'm going to restore this line.